.circleci/dynamic_config.yml

```
@@ -48,7 +48,9 @@ var_6: &only_pull_requests
       only:
         - /pull\/\d+/
 
-var_7: &all_e2e_subsets ['npm', 'esbuild', 'yarn']
+# All e2e test suites
+var_7_subsets: &all_e2e_subsets ['npm', 'esbuild', 'yarn']
+var_7_build_types: &all_e2e_build_types ['e2e', 'snapshot']
```

```suggestion
var_7: &all_e2e_subsets ['npm', 'esbuild', 'yarn']
var_8: &all_e2e_build_types ['e2e', 'snapshot']
```

Since the key in this pair isn't actually used by yaml, we typical just count rather than providing any "context" in the naming scheme.